src/resources/elements/primeDesignSystem/_styles.scss

```diff
@@ -14,39 +14,37 @@
   color: $Neutral04;
   border-radius: $border-radius-normal;
   outline: none;
-  padding: $spacing-normal;
   box-sizing: border-box;
   display: flex;
-  column-gap: $spacing-normal;
   align-items: center;
   height: $input-height;
+  cursor: text;
+  overflow: hidden;
 
-  input {
+  input, textarea {
     color: $Neutral04;
+    height: 100%;
+    padding: $spacing-normal;
 
     &::placeholder {
       color: $Neutral02;
     }
   }
 
+  .fas {
```

Comment:
I prefer that we apply and select our own class that applies specifically to the element(s) being styled rather than referencing a more generic and third-party class.